### PR TITLE
Update unsynced packages

### PR DESCRIPTION
**Pull request by jim-v**

### CODE REVIEW

File: update-magni-image.sh [hunk 1]
```diff
@@ -31,11 +31,8 @@ echo "Installing extra packages"
 sudo apt-get install ros-kinetic-teleop-twist-keyboard
 
 # If current packages are not available via debs, then build from source
-if [[ ! `apt-cache policy ros-kinetic-move-basic` =~ "Installed: 0.2.2" ]]; then
-   get_source 'move_basic'
-fi
-if [[ ! `apt-cache policy ros-kinetic-magni-robot` =~ "Installed: 0.2.1" ]]; then
-   get_source 'magni_robot'
+if [[ ! `apt-cache policy ros-kinetic-ubiquity-motor` =~ "Installed: 0.6.1" ]]; then
+   get_source 'ubiquity-motor'
```
Comment (rohbotics) [inline update-magni-image.sh]: Should be ubiquity_motor

**Review by rohbotics — CHANGES_REQUESTED** [review]
The typo needs fixing.

I am not so sure about this whole system. You will be stuck with a source checkout with ubiquity_motor when the sync actually happens.

I think for this it makes more sense to test out a "patch" debian. If I put a ros-kinetic-ubiquity-motor deb with version 0.6.1~ubiquity1 that should work more seamlessly, and switch back to the OSRF version once it syncs.

File: update-magni-image.sh (hunk 1 above)
Comment (jim-v) [reply update-magni-image.sh]: I agree that giving people source with no easy way to get back to debs is not a good solution. If you can put a deb on packages.ubiquityrobotics.com until the OSRF sync happens, that is much cleaner.

**Comment by rohbotics** [comment]
I built a temporary backport manually. 

https://packages.ubiquityrobotics.com/ubuntu/ubiquity-testing/pool/main/r/ros-kinetic-ubiquity-motor/ros-kinetic-ubiquity-motor_0.6.1-0xenial~tbpo1_armhf.deb

If you change your `/etc/apt/sources.list.d/ubiquity-latest.list` to use `ubuntu/ubiquity-testing` instead of `ubuntu/ubiquity` and then update & upgrade you should get the new deb. When the sync happens, that deb will replace this one.

@jim-v Can you make sure that when you install that deb, the issue goes away (and the motors still work)? I will sync it to `ubuntu/ubiquity` if it works.

**Comment by rohbotics** [comment]
If this process works, I will look into automating it so that we can use it easily with all of our packages.

**Comment by jim-v** [comment]
I removed my source checkout, installed the deb and everything seems good.  
The wheels turn with `teleop_twist_keyboard`, and don't do anything weird when the time is changed.

**Comment by rohbotics** [comment]
Ok, synced to the production repo.

The installed version number from apt-cache is `0.6.1-0xenial~tbpo1`

File: update-magni-image.sh [hunk 2]
```diff
@@ -26,21 +26,4 @@ function get_source {
...
-
-# If current packages are not available via debs, then build from source
-if [[ ! `apt-cache policy ros-kinetic-move-basic` =~ "Installed: 0.2.2" ]]; then
-   get_source 'move_basic'
-fi
-if [[ ! `apt-cache policy ros-kinetic-magni-robot` =~ "Installed: 0.2.1" ]]; then
-   get_source 'magni_robot'
-fi
-
-get_source 'ubiquity_launches'
```
Comment (rohbotics) [inline update-magni-image.sh]: Keep the getting `ubiquity_launches` from source

**Review by rohbotics — CHANGES_REQUESTED** [review]

**Review by rohbotics — APPROVED** [review]

**Pull request merged**